app/Helpers/video_helper.php

```diff
@@ -36,7 +37,7 @@ function embedVideo(string $code, string $width = '640', string $height = '385')
             $code = $result['v'] ?? null;
         }
 
-        if (isset($code) && ! empty($code)) {
+        if (isset($code) && ($code !== '' && $code !== '0' && $code !== [])) {
```

Comment:
I think `in_array()` can be used for it since `$code` is already defined above:

```suggestion
        if (! in_array($code, ['', null, '0', []], true)) {
```

Reply:
Done